django_mailbox/models.py

```diff
@@ -96,6 +98,14 @@ class Mailbox(models.Model):
         default=True,
     )
 
+    last_polling = models.DateTimeField(
+        _(u"Last polling"),
+        help_text=(_("The time of last successfull polling of message. "
```

Comment:
Err, I guess I should've put my above comment here instead, but I'm sure you understand.